demo/js/demo.js

```diff
@@ -475,6 +475,8 @@ window.onload = function() {
 
                 switch (x) {
                     case idx.LAST_NAME:
+                        config.color = config.value != null && (config.value + '')[0] === 'S' ? 'red' : '#191919';
```

Comment:
`value == null` and `value != null` are special cases acceptable to the linter meaning `value === undefined || value === null` and `value !== undefined && value !== null`, respectively.(This was existing code; I just moved it.)